mantidproject/mantid
# ConfigService loading

Users with old `*FilterChannel` properties would fail to get logging configured in mantid. This prints a warning to `std::cerr` and comments out the offending line.

**To test:**

Add a line containing `logging.channels.consoleFilterChannel.level=information` to your properties file and see the new error message. 

Bonus item: Add a line that isn't `key=value` and see mantid fail to start with a proper error message.

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
